### ISSUE TITLE
Feature Stories

### ISSUE THREAD

**Issue by sandersonet** [aptible/diesel.aptible.com#28]
### 1. Apps & databases

- [x] Operations history (infinite scroll?) (#29) (pagination of any kind is blocked by the API not supporting pagination)
- [x] Scale slider, add db type selector (in progress: #33. PR: #37)
- [x] Create disk when creating DB (#38 PR: #39 )

### 2. Roles & permissions

- [x] Verify Stack permissions when CRUD apps/dbs (#42)
  * Users with a 'Privileged' role, can read and manage all stacks inside Org
  * User with Read or Manage permission can see a Stack and its apps and dbs
  * User must have Manage permission to edit stack settings, create/edit apps and dbs

### 3. First-time UX

- [ ] First-time wizard (#30)
  *  Create SSH Key
  * Create App
  * Create Database
  * Configuration details

### 4. Deployment/CI
- [ ] Anything left here?

* * *

### 1. Auth

- [x] Account creation flow
- [ ] Organization creation
- [ ] Stack creation
- [ ] Password reset
- [ ] Invitation acceptance
- [ ] Organization switcher

Account on-boarding flow requires some Auth rework as we will have to move our Stripe integration from dashboard.aptible.com to auth.aptible.com.

### 2. User settings
- [ ] Profile
- [ ] Authentication settings: email/password
- [ ] SSH Key management

### 3. Organization settings
- [ ] Overview
  * Invitations CRUD
  * Member list
  * Billing Contact and Security Officer
  * Alert emails
- [ ] Stack management
- [ ] Role management
- [ ] Organization settings

**Comment by sandersonet** [comment]
Updated to separate required features from optional features that can be handled by existing dashboard if necessary.  Features are roughly ordered by priority.

**Comment by sandersonet** [comment]
Closing in favor of #51
